### PR TITLE
Colin Dismuke - Sprint-Challenge--Testing

### PR DESCRIPTION
**Pull request by cpdis**
@katiagilligan888

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,30 +25,40 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
-1. What is the point of `Test Driven Development`? What do you think about this approach?
-1. Mention three types of automated tests.
+
+    `describe()` creates a block that groups together related tests. `it()` creates a specific test case. `describe()` is useful for grouping together groups of `it()` statements that are related while `it()` is good for testing specific aspects of the code.
```
Comment (katiagilligan888) [inline README.md]: Answers look good 👍

File: api/server.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            let res = await request(server).get("/games");
+            expect(res.status).toBe(200);
+        });
+    });
+
+    it("Respond with an empty array", async () => {
+        let expected = [];
+        res = await request(server).get("/games");
+        expect(res.body).toEqual(expected);
+    });
```
Comment (katiagilligan888) [inline api/server.test.js]: Other GET request tests that you can add: should return an array of data and should return a response of type JSON

File: api/server.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                genre: "FPS",
+                releaseYear: 2004
+            };
+
+            let res = await request(server).post("/games").send(body)
+
+            expect(res.body).toEqual([1])
+        })
+
+        it("Respond with status code 422 when incomplete", async () => {
```
Comment (katiagilligan888) [inline api/server.test.js]: This test can be split into two separate tests: one to test if the title is empty in the request body and another to test if the genre is empty in the request body
